hipify_cli.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import sys
+import argparse
+from hipify import hipify_python
+
+parser = argparse.ArgumentParser(
+    description='Top-level script for HIPifying, filling in most common parameters')
+parser.add_argument(
```

Comment:
I have used similar structure and parameter passing in tensorpipe and it is working. 
LGTM.
Only thing I havent used is --ignores and --includes.